### PR TITLE
Documentation: make "Consul" section more objective

### PR DESCRIPTION
**Pull request by gyuho**
Just minor updates in doc. /cc @philips

### CODE REVIEW

File: Documentation/learning/why.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ When considering features, support, and stability, new applications planning to
 
 ### Consul
 
-Consul bills itself as an end-to-end service discovery framework. To wit, it includes services such as health checking, failure detection, and DNS. Incidentally, Consul also exposes a key value store with mediocre performance and an intricate API. As it stands in Consul 0.7, the storage system does not scales well; systems requiring millions of keys will suffer from high latencies and memory pressure. The key value API is missing, most notably, multi-version keys, conditional transactions, and reliable streaming watches.
+Consul bills itself as an end-to-end service discovery framework. It provides built-in health checking, failure detection, and DNS services. In addition, Consul exposes a key value store with RESTful HTTP APIs. [As it stands in Consul 1.0][dbtester-comparison-results], the storage system does not scale well; systems requiring millions of keys will suffer from high latencies and memory pressure. The key value API is missing, most notably, multi-version keys, conditional transactions, and reliable streaming watches.
```
Comment (philips) [inline Documentation/learning/why.md]: s%bills itself as an%is an%

Comment (philips) [reply Documentation/learning/why.md]: Perhaps "As it stands Consul 1.0 doesn't scale as well as other systems like etcd or Zookeeper in key-value operations"

**Comment by philips** [comment]
Some language suggestions but adding the link is very helpful!

**Comment by gyuho** [comment]
All fixed. Merging in greens and backporting to release branches. Thanks.

**Pull request merged**